docs/src/api/params.md

```diff
@@ -605,6 +605,13 @@ for a list of supported timezone IDs. Defaults to the system timezone.
   - `accuracy` ?<[float]> Non-negative accuracy value. Defaults to `0`.
 
 ## context-option-locale
+* langs: js
```

Comment:
Instead of this, you can just write the description inline in instead of `## property: TestOptions.locale = %%-context-option-locale-%%` in class-testoptions.md, that way it won't affect anything but test runner docs.